db/migrate/20190424145213_rename_file_revision_to_blob_revision.rb

```diff
@@ -0,0 +1,12 @@
+class RenameFileRevisionToBlobRevision < ActiveRecord::Migration[5.2]
```

Comment:
I'd suggest that we do this migration in a separate PR and do the first deploy of the code with it using the old table names but new model names. 

Then we can do another one that flips things over and can break and would be a much smaller amount of changes.

Reply:
I had a quick think about possible approaches, but as it quickly became apparent the change is a breaking one, it seemed more valuable to keep everything in one PR to aid understanding. As changing table names should be fast, is there really much value in separating it?

Reply:
Yup there is a breaking part of it whatever happens. What I'm encouraging is minimising the amount that is in the breaking piece of work.

Thus you'd have as much as possible in something that can be merged in as normal and avoids any special deployment considerations and is pretty safe to undo. Then you'd have a change that is db migration and model changes that needs to go from merge to deploy as quick as possible and is rather painful to undo.

Reply:
How about now? (I didn't think this was doable for the column renames initially.)